sdk/examples/xo_python/sawtooth_xo/xo_client.py

```diff
@@ -35,9 +36,10 @@ def _sha512(data):
 
 
 class XoClient:
-    def __init__(self, base_url, keyfile):
+    def __init__(self, base_url, keyfile, user):
```

Comment:
What will the value of `user` be if not specified in the config file? Should this parameter default o `None`?